source/examples/rapids-autoscaling-multi-tenant-kubernetes/notebook.ipynb

```diff
@@ -222,9 +222,9 @@
     "In order to steam the container image to the GKE nodes our image needs to be stored in [Google Cloud Artifact Registry](https://cloud.google.com/artifact-registry/) in the same region as our cluster.\n",
     "\n",
     "```console\n",
-    "$ docker pull rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10\n",
+    "$ docker pull rapidsai/base:24.04-cuda11.8-py3.10\n",
     "\n",
-    "$ docker tag rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10 REGION-docker.pkg.dev/PROJECT/REPO/IMAGE:TAG\n",
+    "$ docker tag rapidsai/base:24.04-cuda11.8-py3.10 REGION-docker.pkg.dev/PROJECT/REPO/IMAGE:TAG\n",
```

Comment:
Similar to https://github.com/rapidsai/deployment/pull/379/files#r1626435519, this is another oudated image reference I noticed.

Will look for them more systematically in a follow-up PR. But some of the references have been out of date for a while, so aren't critical for the 24.06 release.